<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openBook">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить список книг(*.txt)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openReader">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить список читателей(*.txt)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>381</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>381</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>380</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generatePromiser">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать список выданных книг</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>360</width>
      <height>71</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Поиск книги</string>
    </property>
    <widget class="QPushButton" name="findBookByAuthor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>По Ф.И.О. автора</string>
     </property>
    </widget>
    <widget class="QPushButton" name="findBookByName">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>По названию</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>381</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск читателя по Ф.И.О</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generatePromiser_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>381</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить список должников</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>360</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Сортировать книги</string>
    </property>
    <widget class="QPushButton" name="NumberOfBookSort">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>По номеру</string>
     </property>
    </widget>
    <widget class="QPushButton" name="YearSort">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>По году издания</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="showListOfBooks">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать список</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showListOfReaders">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать список</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newReader">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить читателя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteReader">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить читателя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newBook">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить книгу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBook">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить книгу</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
